src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -1364,4 +1517,12 @@ public void fieldCollapsingTest() throws IOException {
     private String getScrollId(JSONObject response) {
         return response.getString("_scroll_id");
     }
+
+    private void checkResponseSize(JSONObject response, int sizeCheck, boolean isGroupBy) {
+        JSONArray queryResponse = getHits(response);
+        if (isGroupBy) {
```

Comment:
instead of parameter please introduce 2 methods with names like

checkReponseSize
checkAggregationReponseSize